opentyrian.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted upstream.

Reply:
I already tried. They don't want it https://github.com/opentyrian/opentyrian/pull/13.